SEQLinkage/Core.py

```diff
@@ -272,11 +296,25 @@ def extract_vcf_with_anno(self,data):
                         data.famvarmafs[k].append(maf if maf < 0.5 else 1-maf)
                         for person, g in zip(data.families[k], gs):
                             data[person].append(g if maf<0.5 else self.reverse_genotypes(g))
-            data.variants.append([self.vcf.GetChrom(), self.vcf.GetPosition(), self.name]) #remove maf
+            data.variants.append([self.vcf.GetVariantID(), self.vcf.GetPosition(), self.name]) #remove maf
             #print(i,varmafs.shape,self.chrom, self.startpos, self.endpos, self.name,self.vcf.GetPosition())
             varIdx += 1
         return varIdx
 
+    def check_gs(self,gs):
+        '''skip monomorphic variants and signton variants in a family'''
```

Comment:
"singleton" is a typo.

Reply:
Make sure that missing genotype is properly handled here too.

Reply:
updated